src/model.ts

```diff
@@ -523,23 +530,29 @@ export class Model extends EventBus<any> implements CommandDispatcher {
         }
         this.status = Status.Running;
         const { changes, commands } = this.state.recordChanges(() => {
+          const start = performance.now();
+          console.group(`Dispatching of ${type}`);
```

Comment:
I feel like logging every single command is too much.
Something as simple as scrolling spams the console :/

Maybe log it only if it took more than X ms ?